src/components/auth/sign-up.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable react/jsx-indent */
+import { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+
+import AuthForm from './authform';
+
+const SignUp = () => {
+  const navigate = useNavigate();
+  const endpoint = 'https://sheltered-tor-84017.herokuapp.com/api/v1/users/register';
+  const userState = useSelector((state) => state.user);
+
+  const onSignUp = () => {
+    navigate('/login');
+  };
```

Comment:
Also, it'd be nice to have a notification when the user is signed up.
The backend already returns a response, you should go ahead and display that. There's no need to create your own message since this will make your work faster